cmd/cstor-pool-mgmt/controller/replica-controller/handler.go

```diff
@@ -650,7 +650,12 @@ func (c *CStorVolumeReplicaController) syncCVRStatus(cvr *apis.CStorVolumeReplic
 
 	err = volumereplica.GetAndUpdateSnapshotInfo(c.clientset, cvr)
 	if err != nil {
-		return errors.Wrapf(err, "unable to update snapshot list details in CVR")
+		c.recorder.Event(
```

Comment:
https://github.com/openebs/maya/pull/1675/files#diff-0ae3c49c30165eb14a67389383ec46f8R645 this one also.

Reply:
Updated